lib/BridgeAbstract.php

```diff
@@ -142,7 +155,10 @@ public function setDatas(array $inputs){
 			if($time !== false
 			&& (time() - static::CACHE_TIMEOUT < $time)
 			&& (!defined('DEBUG') || DEBUG !== true)){
-				$this->items = $this->cache->loadData();
+				$cached = array();
```

Comment:
In didn't understand your question or I didn't understand what I've done.
If we save items + extraInfos in cache, we should set items + extraInfos back from cache.